simulationdataschema/model_system.py

```diff
@@ -207,16 +209,20 @@ def get_geometric_space_for_atomic_cell(self, logger):
         self.volume = cell.volume * ureg.angstrom**3
 
     def normalize(self, archive, logger) -> None:
-        # We try to get ASE atoms from the cache of AtomicCell
-        ase_atoms = self.to_ase_atoms(logger)
-        if ase_atoms is not None:
+        # Skip normalization for `Entity`
+        try:
+            ase_atoms = self.to_ase_atoms(logger)  # function defined in AtomicCell
             self.get_geometric_space_for_atomic_cell(logger)
+        except Exception as e:
+            logger.warning(
+                "Could not extract the geometric space information from ASE Atoms object.",
+            )
+            return
```

Comment:
`try-except` is indeed more Pythonic.

> except Exception as e:

Try to nail down the exception type as much as possible. Just leaving it at `Exception` is typically considered bad practice, since you might be catching genuine bugs too.
Moreover, if you don't use `e`, then don't extract it.